server/runtime.go

```diff
@@ -37,16 +40,124 @@ func (s *Server) Version(ctx context.Context, req *pb.VersionRequest) (*pb.Versi
 func (s *Server) CreatePodSandbox(ctx context.Context, req *pb.CreatePodSandboxRequest) (*pb.CreatePodSandboxResponse, error) {
 	var err error
 
-	// TODO: Parametrize as a global argument to ocid
-	ocidSandboxDir := "/var/lib/ocid/sandbox"
-	podSandboxDir := filepath.Join(ocidSandboxDir, req.GetConfig().GetName())
+	if err := os.MkdirAll(s.sandboxDir, 0755); err != nil {
+		return nil, err
+	}
 
+	// process req.Name
+	name := req.GetConfig().GetName()
+	var podSandboxDir string
+
+	if name == "" {
```

Comment:
Yeah. I thought so. 
But inside `api.pb.go`, those json tags are `omitempty`. So I am not 100% sure.